apps/libexec/slim-poller.py

```diff
@@ -0,0 +1,146 @@
+import fcntl, sys, os
+import subprocess as sp
+import shlex
+
+
+LOCK_FILE = "/tmp/slim-poller.lock"
+ENCRYPTION_KEY_PATH = "/opt/monitor/op5/merlin/"
```

Comment:
Would be great if we grab the `merlin_conf_dir` here similar to how it's done in `merlinkey.py.in`